pyresample/ewa/dask_ewa.py

```diff
@@ -107,7 +111,8 @@ def _chunk_callable(x_chunk, axis, keepdims, **kwargs):
 
 def _combine_fornav(x_chunk, axis, keepdims, computing_meta=False,
                     maximum_weight_mode=False):
-    if isinstance(x_chunk, tuple) and len(x_chunk) == 2 and isinstance(x_chunk[0], tuple):
+    is_empty_chunk = isinstance(x_chunk, tuple) and len(x_chunk) == 2 and isinstance(x_chunk[0], tuple)
```

Comment:
Should this be refactored into its own function for clarity?

Reply:
:thinking: it probably could. I'll see what I have time for. Thanks.